### ISSUE TITLE
Add form labels for newsletter input

### ISSUE THREAD

**Issue by SohamJuneja** [SohamJuneja/OpenSourceDSC#12]
The email input in the newsletter form on index.html lacks a proper label, making it inaccessible and failing basic accessibility requirements

**Comment by LSUDOKO** [comment]
can you assign me this issue

**Comment by SohamJuneja** [comment]
You already have an issue assigned @LSUDOKO
